.eslintrc.json

```diff
@@ -28,7 +28,6 @@
     ],
     "no-underscore-dangle": 0,
     "import/no-mutable-exports": 1,
-    "jsx-a11y/anchor-is-valid": "off",
```

Comment:
@nkpremices why though?

Reply:
This was for solving the issue with `<Link />` components @MemunaHaruna

Reply:
It was asking for a `href` props instead of a `to` props. @MemunaHaruna